frontend/www/js/omegaup/components/course/StudentProgress.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <tr>
+    <td class="text-center align-middle">
+      <a v-bind:href="studentProgressUrl()">
+        {{ student.name || student.username }}
+      </a>
+    </td>
+    <td
+      class="score flex-column justify-content-center align-items-center"
+      v-for="assignment in assignments"
+    >
+      <p class="mb-1 text-center">{{ Math.round(score(assignment)) }}%</p>
+      <div class="d-flex justify-content-center">
+        <div v-if="student.progress.hasOwnProperty(assignment.alias) == false">
+          {{ T.wordsProblemsUnsolved }}
+        </div>
+        <div
+          v-for="problem in student.progress[assignment.alias]"
```

Comment:
nit:

```suggestion
          v-for="problemScore in student.progress[assignment.alias]"
```

given that this is not a problem, but the problem's score, it might be better to call this `problemScore`.